aws-encryption-sdk-net/Source/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+using System.Reflection;
+
+[assembly: AssemblyTitle("AWS.EncryptionSDK")]
+
+// This should be kept in sync with the version number in AWSEncryptionSDK.csproj
```

Comment:
Modern .NET versions allow to generate this assembly metadata automatically (e.g. derive `AssemblyVersion` from `Version`), so that we wouldn't need to manually keep these in sync. But so long as we support .NET Framework, AFAICT there isn't an automated mechanism to avoid this duplication.